src/components/Icon/BrandAssets.js

```diff
@@ -0,0 +1,6 @@
+import ExpensifyCashLogo from '../../../assets/images/expensify-cash.svg';
```

Comment:
Should the other assets like the welcome screenshots be included here? This file is useless currently.

Reply:
The welcome screenshots are .pngs/.jpgs as an FYI. I think we'll potentially have more instances in the future of where we need to show pngs/jpgs instead of svgs, so just something to consider.

Reply:
> This file is useless currently.

That's not 100% true, because in `src/components/Icon/index.js`, `Expensicons` are given a default fill color, while `BrandAssets` are not. That's because brand assets like the E-cash circle logo have their fill colors predefined in their `.svg` files, while `Expensicons` do not. So these files, `Icon/BrandAssets.js` and `Icon/Expensicons.js`, serve the purpose of categorizing the `.svg` assets we have into brand assets and expensicons.

Reply:
Oh, actually I see what you mean /w this file only having one import. I think the important question @shawnborton is whether or not there will be more full-color `.svg` assets in the future.

Reply:
I think the answer is probably. These scenarios aren't yet clearly defined, but some potential places where they _could_ show up:
- if we ever have any kind of empty state illustration
- if we ever reuse the inbox choice card pattern that has the multi-colored icons
- if we ever have custom emojis

Reply:
OK, thanks for the context! I think this sounds exactly like a case of over-engineering then. For the one place that uses `expensify-cash.svg`, I suggest importing it directly and then simplifying all the logic around brand-assets not needing fills applied to them.